tests/test_lithium.py

```diff
@@ -18,7 +18,7 @@
 import tempfile
 import unittest
 
-import lithium  # pylint: disable=relative-import
+from lithium import reducer
```

Comment:
This works, but if you do `import lithium`, you can use `lithium...` throughout and the final diff will be much smaller. I'll leave it up to you, but it has the advantage that `lithium.Lithium` is also the API used elsewhere.

Reply:
Alright, I'll synchronise to `lithium.Lithium`.